### PR TITLE
Update link preview test cases

### PR DESCRIPTION
**Pull request by jasonblais**
#### Summary
Update link preview test cases

### CODE REVIEW

File: tests/test-link-preview.md [hunk 1]
```diff
@@ -14,64 +14,38 @@ Post location variation:
...
-#### C) Wikipedia 
-
-Link 1 example: https://en.wikipedia.org/wiki/Olympus_Mons
-
-Link 2 example: https://en.wikipedia.org/wiki/Climate_of_Mars#Effect_of_dust_storms
-
-Link 3 example: https://en.wikipedia.org/wiki/Mars_Exploration_Rover
+#### B) Link preview without an image
 
+Link 1 example: coveralls.io/builds/9818822/source?filename=app%2Faudit.go
```
Comment (esethna) [inline tests/test-link-preview.md]: This link doesn't render for me 
image

File: tests/test-link-preview.md [hunk 2]
```diff
@@ -14,64 +14,38 @@ Post location variation:
...
 
 ```
-#### D) YouTube 
-
-Link 1 example: https://www.youtube.com/watch?v=fDu1S-MMClw
-
-Link 2 example: https://www.youtube.com/watch?v=GaCjjda5v1o
-
-Link 3 example: https://www.youtube.com/watch?v=1Ui44_-N9y0
+#### C) Link preview with image in top right corner
```
Comment (esethna) [inline tests/test-link-preview.md]: These links only show the image if you click expand. And the image appears below the preview text:

image

File: tests/test-link-preview.md [hunk 3]
```diff
@@ -14,64 +14,38 @@ Post location variation:
...
-#### E) GitHub
+Link 3 example: https://github.com/mattermost
 
-Link 1 example: https://github.com/mattermost
+Link 4 example: http://stackoverflow.com/questions/36650437/using-mattermost-api-via-gitlab-oauth-as-an-end-user-with-username-and-password
 ```
 
 ```
-#### F) Webpage: Techmeme 
+#### D) Link preview with image at the bottom
```
Comment (esethna) [inline tests/test-link-preview.md]: Only shows if you click the expand arrows. Is this expected?

File: tests/test-link-preview.md (hunk 1 above)
Comment (jasonblais) [inline tests/test-link-preview.md]: sorry, it was missing the `https` part. Will fix

File: tests/test-link-preview.md (hunk 3 above)
Comment (jasonblais) [inline tests/test-link-preview.md]: I can add that to the intro

File: tests/test-link-preview.md (hunk 2 above)
Comment (jasonblais) [inline tests/test-link-preview.md]: Which OS/browser?

This is the expected and how they render for me

image

**Pull request merged**